<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFileRenameTools</class>
 <widget class="QWidget" name="QFileRenameTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>根目录：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="srcRoot"/>
     </item>
     <item>
      <widget class="QPushButton" name="srcBrowse">
       <property name="text">
        <string>browse</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>输出目录：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dstRoot"/>
     </item>
     <item>
      <widget class="QPushButton" name="dstBrowse">
       <property name="text">
        <string>browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>包含关键字:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="has"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>不包含关键字:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nothas"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>后缀:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="suffix"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QListWidget" name="srcList"/>
     </item>
     <item>
      <widget class="QListWidget" name="dstList"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>字符串替换:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="srcStr"/>
     </item>
     <item>
      <widget class="QLineEdit" name="dstStr"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>生成格式:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="format"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="preview">
       <property name="text">
        <string>预览</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="general">
       <property name="text">
        <string>生成</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
